masu/util/__init__.py

```diff
@@ -1 +1,51 @@
-# noqa
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Masu utility methods."""
+
+import os
+
+import watchtower
+from boto3.session import Session
+
+
+def setup_cloudwatch_logging(logger):
+    """Add Cloud Watch log handler if appropriate."""
+    key_id = os.environ.get('CW_AWS_ACCESS_KEY_ID')
+    secret = os.environ.get('CW_AWS_SECRET_ACCESS_KEY')
```

Comment:
Is there some reason you aren't using the standard AWS creds environment variables?

Also, is there some reason you're pulling directly from `os.env` and not routing through the Config object?

Reply:
This is a different AWS key/secret than ours. This is from the platforms key/secret to feed data into their Kibana.

I can move it to the config.

Reply:
Please also include documentation that discusses this detail.